### DISCUSSION TITLE
midwayjs  bull-board 要如何自定义访问链接

### DISCUSSION THREAD

**Discussion by mynamexjl** [midwayjs/midway#3706] [Q&A]
### Describe the problem（描述问题）

不想使用默认链接访问的话，可以在那边配置。

### Midway Versions（Midway 版本）

    "@midwayjs/bull": "3.11.6",
    "@midwayjs/bull-board": "^3.11.6",

**Comment by czy88840616** [comment]
https://github.com/midwayjs/midway/blob/main/packages/bull-board/src/configuration.ts#L17 这个配置
